pkgs/os-specific/linux/numactl/default.nix

```diff
@@ -11,6 +11,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-ry29RUNa0Hv5gIhy2RTVT94mHhgfdIwb5aqjBycxxj0=";
   };
 
+  patches = [
+    # Fix for memory corruption in set_nodemask_size
+    (fetchpatch {
+      url = "https://github.com/${pname}/${pname}/commit/f9deba0c8404529772468d6dd01389f7dbfa5ba9.patch";
```

Comment:
I'd suggest avoiding `pname` in fetchers.

See https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Thank you! I applied your suggestion and I’ll keep that in mind for future submissions.